raiden/routing.py

```diff
@@ -36,6 +36,32 @@ def get_best_routes(
 ) -> List[RouteState]:
     services_config = config.get("services", None)
 
+    if to_address in views.all_neighbour_nodes(chain_state):
+        neighbours = get_best_routes_internal(
+            chain_state=chain_state,
+            token_network_id=token_network_id,
+            from_address=from_address,
+            to_address=to_address,
+            amount=amount,
+            previous_address=previous_address,
+        )
+        channel_state = views.get_channelstate_by_token_network_and_partner(
+            chain_state=chain_state,
+            token_network_id=token_network_id,
+            partner_address=Address(to_address),
+        )
+
+        for route_state in neighbours:
+            if to_address == route_state.node_address:
+                if (
+                    channel_state
+                    and channel.get_distributable(
```

Comment:
Please add a comment describing that the other conditions (channel is open, ...) are already checked by `get_best_routes_internal`